conjure/types/types.go

```diff
@@ -326,3 +326,16 @@ func (f *funcType) ImportPaths() []string {
 	}
 	return importPaths
 }
+
+func MapBinaryTypeToReadCloserType(valType Typer) Typer {
```

Comment:
Can you add a comment that explains what that this function does?

Also, it doesn't look like there's an effective way to tell if a read closer type was chosen here. Might make sense to update the API to return the `Typer` and any imports that are needed for the typer and then to add the `io` package to the return value if the `IOReadCloserType` or its variant is selected. That way, callers that need the imports can just add the returned imports after calling this.